stripe/src/main/java/com/stripe/android/view/LockableHorizontalScrollView.java

```diff
@@ -50,4 +68,41 @@ public boolean onInterceptTouchEvent(MotionEvent event) {
         // Reject all intercept events if mScrollable is false.
         return mScrollable && super.onInterceptTouchEvent(event);
     }
+
+    void setScrollChangedListener(LockableScrollChangedListener listener) {
```

Comment:
For testing, but I suppose you could want to listen for other reasons. It doesn't harm anything if you're being nosy.